common/fixation_entry.go

```diff
@@ -301,8 +301,9 @@ func (fs *FixationStore) FindEntry(ctx sdk.Context, index string, block uint64,
 func (fs *FixationStore) GetEntry(ctx sdk.Context, index string, entryData codec.ProtoMarshaler) bool {
 	safeIndex, err := types.SanitizeIndex(index)
 	if err != nil {
-		details := map[string]string{"index": index}
-		utils.LavaError(ctx, ctx.Logger(), "GetEntry_invalid_index", details, "invalid non-ascii entry")
+		utils.LavaFormatError("invalid non-ascii entry", err,
```

Comment:
description string is a duplicate in all of the functions, change it to give us an idea where it came from

Reply:
done